### PR TITLE
COR-149: add registration status label to invididual list page

### PR DESCRIPTION
**Pull request by shinroo**
### Original ticket description:

As a { Project Manager | Enumerator | Triage Staff | Expert }
I want to see a beneficiaries status from the beneficiary list page
So that I can take pertinent actions for this beneficiary

Technical Description:

Beneficiary entry should show status. If status is an action, status should be a button pointing to form.

### Note

Since we did not yet go down the route of using actions, the later part of the requirement:

> If status is an action, status should be a button pointing to form.

Has not been implemented.

### CODE REVIEW

**Comment by shinroo** [comment]
Screenshot from 2021-08-09 15-45-39

File: api/pkg/apps/webapp/individuals.go [hunk 1]
```diff
@@ -48,9 +54,23 @@ func (s *Server) Individuals(w http.ResponseWriter, req *http.Request) {
...
+			RegistrationStatus: rc.Status(),
+		})
+	}
+
 	if err := s.renderFactory.New(req, w).ExecuteTemplate(w, "individuals", map[string]interface{}{
-		"Individuals": list,
-		"Page":        "list",
+		"Individuals":             list,
+		"IndividualsWithStatuses": beneficiaryWithStatuses,
+		"Page":                    "list",
```
Comment (nilueps) [inline api/pkg/apps/webapp/individuals.go:73]: Is it necessary to pass in an entirely new pipe?

```
type Individual struct {
    *iam.Individual
    Status *registrationctrl.Status
}
```

Then

```
{{range .Individuals}}
    ...
    {{if .Status}}...{{end}}
{{end}}
```

**Review by nilueps — COMMENTED** [review]
This works, but I think you can simplify the handler a little, see comment

File: api/pkg/apps/webapp/individuals.go (hunk 1 above)
Comment (shinroo) [reply api/pkg/apps/webapp/individuals.go:73]: Currently, `Individuals` is an individual list.

The pagination relies on that in the following code segment:
```
{{template "pagination-navigation" .Individuals.Metadata }}
```

I had initially tried to just pass a map of `id` -> `registrationctrl.Status` and then simply call that from the individual row. But I wasn't able to get that working.

Comment (nilueps) [reply api/pkg/apps/webapp/individuals.go:73]: Ah right right, well in that case I think your solution is fine

**Review by nilueps — APPROVED** [review]

**Pull request merged**